pymongo/ssl_support.py

```diff
@@ -28,6 +30,25 @@
         import pymongo.ssl_context as _ssl  # type: ignore[no-redef]
     except ImportError:
         HAVE_SSL = False
+except AttributeError:
```

Comment:
Could you refactor this into the `except ImportError:` block so we don't have to duplicate the stdlib ssl fallback?

```python
except (ImportError, AttributeError) as exc:
    if isinstance(exc, AttributeError):
        warnings.warn(...)
```

Reply:
Excellent idea. I still kept the fallback warning as it seems appropriate for the other exception, too.